### ISSUE TITLE
what do i need to do to run branch 5.x successfully?

### ISSUE THREAD

**Issue by WayThought** [SDWebImage/SDWebImage#2654]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 11.0 / 10.13.0 / 11.0 / 4.0
 SDWebImage Version      | e.g. 4.2.0 / 4.1.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 9 / Xcode 8
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by WayThought** [comment]
error: /Users/qianguojie/Desktop/test/SDWebImage/Examples/Pods/Target Support Files/Pods-SDWebImage Watch Demo Extension/Pods-SDWebImage Watch Demo Extension.debug.xcconfig: unable to open file (in target "SDWebImage Watch Demo Extension" in project "SDWebImage Demo") (in target 'SDWebImage Watch Demo Extension')

error: /Users/qianguojie/Desktop/test/SDWebImage/Examples/Pods/Target Support Files/Pods-SDWebImage Watch Demo Extension/Pods-SDWebImage Watch Demo Extension.debug.xcconfig: unable to open file (in target "SDWebImage Watch Demo Extension" in project "SDWebImage Demo") (in target 'SDWebImage Watch Demo Extension')

**Comment by dreampiggy** [comment]
Make sure you're using CocoaPods 1.5.3 - 1.6.1 (any version is OK, suggested using 1.6.1). Following the step of command under the repo root directory...

```bash
cd Example
pod install
cd ../Tests
pod install
open ../*.xcworkspace
```

If you're using Xcode 10.2, please wait this MR been merged to solve the compatible for Xcode 10.2. #2652

**Comment by dreampiggy** [comment]
Need we add some step toturial to the Wiki pages for the contributor or user who want to build the total project ?

For normal manual install framework, actually, you don't need to open `SDWebImage.xcworkspace`. Instead, open `SDWebImage.xcodeproj` and build to produce the frameworks. The workspace contains the both Example && Test Application used by our contributors.

**Comment by WayThought** [comment]
$ pod --version
1.5.3

Go directly to "/Users/( your name)/.cocoapods/repos/master/Specs/1/9/2/libwebp" ,select1.0.2folder.
Open libwebp.podspec.json.
Change git address to https://github.com/webmproject/libwebp.git

Then follow your steps and just fine.

**Comment by WayThought** [comment]
$ pod --version 1.5.3
I go directly to "/Users/( your name)/.cocoapods/repos/master/Specs/1/9/2/libwebp" ,select 1.0.2 folder.  Open libwebp.podspec.json.  Change git address to https://github.com/webmproject/libwebp.git
Then follow your steps and just fine.

**Comment by dreampiggy** [comment]
Why need this ?

> $ pod --version
>  1.5.3
> I go directly to "/Users/( your name)/.cocoapods/repos/master/Specs/1/9/2/libwebp" ,select 1.0.2 folder.
>  Open libwebp.podspec.json.  Change git address to https://github.com/webmproject/libwebp.git
> Then follow your steps and just fine.

Why need to modify the libwebp podspec source ? It's worked for all the pod version.

**Comment by kinarob** [comment]
@WayThought Now, @dreampiggy Fix this, you can run branch 5.x successfully.
